src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -82,14 +82,21 @@ public void updateLimelightPoseEstimate(int limelightNumber) {
     if (headingRateDegreesPerSecond < VisionConstants.MEGA_TAG_2_MAX_HEADING_RATE) {
       LimelightHelpers.SetRobotOrientation(
           getLimelightName(limelightNumber), headingDegrees, 0, 0, 0, 0, 0);
-      limelightEstimates[limelightNumber] = getMegaTag2PoseEstimate(limelightNumber);
+      return limelightEstimates[limelightNumber] = getMegaTag2PoseEstimate(limelightNumber);
     } else {
-      limelightEstimates[limelightNumber] = getMegaTag1PoseEstimate(limelightNumber);
+      return limelightEstimates[limelightNumber] = getMegaTag1PoseEstimate(limelightNumber);
     }
     //   }
     // }
   }
 
+  public void updatePoseEstimate(int limelightNumber) {
+    limelightEstimates[limelightNumber] =
+        DriverStation.isEnabled()
```

Comment:
why are we only using mt2 when the drive station isn't enabled?

Reply:
we only use mt1. this is to get an initial pose for the odometry to zero to. 



i think.

Reply:
wait why aren't we using mt2?

Reply:
to get an initial pose that isn't in some rando place. the mt2 estimate will be in a random location unless the pose estimator has a known rotation. this is basically the same thing we were doing with just like setting the last seen pose to the mt1 pose with a button and/or on disable exit. I kinda like this way better just because.

Reply:
isn't your ternary operator backwards then? Right now this will do only MT1 when its enabled.

Reply:
fuck